env/jx-app-ui/values.yaml

```diff
@@ -1,10 +1,13 @@
 jxui:
   enableTLS: true
+  auth:
+    # auth needs to be disabled in JXUI backend when running oauth2-proxy with
+    # GitHub IDP in front of it because JXUI backend treats every bearer token
+    # as JWT while GitHub provides as plain OAuth2 Access Token.
+    # TODO: Set to 'none' only after we're sure that the old Ingress is removed
+    type: sso
```

Comment:
This requires a 2nd PR to be sure in order to disable jxui backend auth only after its Ingress has been deleted.

Reply:
Set to 'none' in this PR since the corresponding Ingress has been deleted manually.